.github/workflows/fossa.yml

```diff
@@ -0,0 +1,27 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "^1.14.x"
+      - run: go version
+      # Runs a set of commands to initialize and analyze with FOSSA
+      - name: run FOSSA analysis
+        env:
+          # FOSSA Push-Only API Token
+          FOSSA_API_KEY: '304657e2357ba57b416b94e6b119131b'
```

Comment:
Sorry if this has been asked before, but shouldn't this be set in an env var, as a secret?

Reply:
@jpkrohling it can't be set as a secret because it won't be passed as a variable when someone will submit a PR from the fork (GitHub Actions do not pass secrets from forks).

This is an API token that can be safely exposed publicly (it was _designed_ to be exposed publicly) - https://docs.fossa.com/docs/api-reference#push-only-api-token.